edx_proctoring/urls.py

```diff
@@ -116,6 +116,11 @@
         views.ProctoredExamAttemptView.as_view(),
         name='proctored_exam.exam_attempts'
     ),
+    url(
+        r'edx_proctoring/v1/settings/exam_id/(?P<exam_id>\d+)/$',
```

Comment:
sorry for a last minute comment. Looking at all the other endpoints we called during an exam the URL path is prefixed with `proctored_exam` can we be more closely copy these endpoints? Something like this:
```suggestion
        r'edx_proctoring/v1/proctored_exam/settings/exam_id/(?P<exam_id>\d+)/$',
```

Reply:
Thanks, I like your suggestion! Done.